ChegeBryan/politico
# #163846710 Refactor code.

#### What does this PR do?

Refactor methods and function on model files and app file.

#### Description of Task to be completed?

- [x] Change class methods to static methods 
- [x] Register app error handlers from a different file

#### How should this be manually tested?
After cloning the repository to your local machine - check
[README.md](README.md) for instructions checkout to `ch-refactor-code-163846710` branch. [Install the requirements](https://github.com/ChegeBryan/politico#installing) and launch the flask server check [here](https://github.com/ChegeBryan/politico#starting-the-server) for instructions. Run the [tests](https://github.com/ChegeBryan/politico#running-the-tests).

#### Any background context you want to provide?
Refactors code for improvement on code duplication highlighted by Code climate.

#### What are the relevant pivotal tracker stories?
[#163846710](https://www.pivotaltracker.com/story/show/163846710)
